estate/models/real_estate.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
+from datetime import datetime, timedelta
+
+class RealEstate(models.Model):
+    _name = "real.estate"
+    _description = "Test model"
+
+    name = fields.Char(default="House",required = True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_available = fields.Date(copy=False, default=lambda self: datetime.today() + timedelta(days=90))
+    expected_price = fields.Float(required = True)
+    selling_price = fields.Float(readonly = True, copy = False)
+    bedrooms = fields.Integer(default = 2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')])
+    active = fields.Boolean(default=True, invisible=True)
+    state = fields.Selection(
+        string='State',
+        selection=[('new', 'New'), ('sold', 'Sold'), ('offer received', 'Offer Received'), ('offer accepted', 'Offer Accepted'), ('canceled', 'Canceled')],
```

Comment:
The first element of the tuple is the technical name, it should respect Python naming conventions
```suggestion
        selection=[('new', 'New'), ('sold', 'Sold'), ('offer_received', 'Offer Received'), ('offer_accepted', 'Offer Accepted'), ('canceled', 'Canceled')],
```